### PR TITLE
Labeled more symbols

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

File: asm/code_80428A0.s [hunk 1]
```diff
@@ -1368,7 +1368,7 @@ _08043594:
 	bl sub_806B6C4
 	b _080435D4
 	.align 2, 0
-_080435B4: .4byte gUnknown_80F5668
+_080435B4: .4byte gCurrentMusic
```
Comment (SethBarberee) [inline asm/code_80428A0.s:1371]:
```suggestion
_080435B4: .4byte gDungeonBGMusic
```

Feel like that is more descriptive given it's an array of each BG Music # corresponded to each dungeon

File: asm/code_8075BA4.s [hunk 2]
```diff
@@ -27364,8 +27364,8 @@ _080835FA:
 	bx r1
 	thumb_func_end sub_8083568
 
-	thumb_func_start sub_8083600
-sub_8083600:
+	thumb_func_start GetLeader
```
Comment (SethBarberee) [inline asm/code_8075BA4.s:27367]:
```suggestion
	thumb_func_start GetLeaderEntity
```

Returns a `DungeonEntity`. Up to you on whether this better or not.

File: asm/code_80428A0.s (hunk 1 above)
Comment (AnonymousRandomPerson) [reply asm/code_80428A0.s]: I went for `gDungeonMusic`.

File: asm/code_8075BA4.s (hunk 2 above)
Comment (AnonymousRandomPerson) [reply asm/code_8075BA4.s]: Done.

**Pull request merged**